### PR TITLE
CPDB 23prelim

### PR DESCRIPTION
**Pull request by fvankrieken**
Tweaks for #460

### CODE REVIEW

File: dcpy/library/script/fisa_dailybudget.py [hunk 1]
```diff
@@ -72,6 +79,17 @@ def ingest(self) -> pd.DataFrame:
         df = self._dedupe(df)
         return df.sort_values(by="TBL_LAST_DT")
 
+    def ingest(self) -> pd.DataFrame:
```
Comment (damonmcc) [inline dcpy/library/script/fisa_dailybudget.py:82]: since this one uses `previous_version`, should this be `ingest_historical`?

Comment (fvankrieken) [reply dcpy/library/script/fisa_dailybudget.py:82]: This is meant to be just the standard ingest. Ingest historical was meant if you had the entire historical archive of files to process. Maybe not the clearest lingo

Comment (fvankrieken) [reply dcpy/library/script/fisa_dailybudget.py:82]: I can add some helpful comments/docstrings

Comment (damonmcc) [reply dcpy/library/script/fisa_dailybudget.py:82]: gotcha. and looks like `ingest_historical` isn't called in `runner`, so someone would have to edit this file to use `ingest_historical`?

Comment (fvankrieken) [reply dcpy/library/script/fisa_dailybudget.py:64]: And by that I mean delete the now-defunct function

**Review by damonmcc — APPROVED** [review]

**Pull request merged**